R/FDAFeatWavelets.R

```diff
@@ -0,0 +1,33 @@
+#' @title Discrete Wavelet transform features.
+#'
+#' @description
+#' The function extracts discrete wavelet transform coefficients from the raw
+#' functional data.
+#'
+#' @param data [\code{data.frame}]\cr
+#'   Functional data.
+#' @param target [\code{character}]\cr
+#'   Name of the target variable. Default: \dQuote{NULL}. The variable is only
+#'   set to be consistent with the API.
+#' @param filter,boundary [\code{character}]\cr
```

Comment:
It is one for each. This is just a way of saving "typing" time since they have the same description.